Spigot-Server-Patches/0760-Add-custom-destroyerIdentity-int-parameter-to-the-se.patch

```diff
@@ -0,0 +1,38 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheFruxz <cedricspitzer@outlook.de>
+Date: Tue, 15 Jun 2021 12:20:44 +0200
+Subject: [PATCH] Add custom destroyerIdentity (int) parameter to the
+ sendBlockBreak(foo:bar) function
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index cfe6898dc373fe55a08acf5c90e200061aa7d0fc..98d5945140912e75c5a915823ea5e9c37cf6db10 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -137,6 +137,7 @@ import org.bukkit.plugin.messaging.StandardMessenger;
+ import org.bukkit.scoreboard.Scoreboard;
+ 
+ import net.md_5.bungee.api.chat.BaseComponent; // Spigot
++import org.jetbrains.annotations.NotNull;
```

Comment:
Don't import here, nullability annotations aren't needed for the craftbukkit impl of the API, so you can either just remove it, or use the fully qualified name below.